templates/rad-lab/gen-ai/main.tf

```diff
@@ -178,14 +178,16 @@ resource "google_service_account_iam_member" "sa_ai_notebook_iam" {
   service_account_id = google_service_account.sa_p_workbench.id
 }
 
-resource "null_resource" "ai_workbench_usermanaged_provisioning_state" {
-  for_each = toset(google_notebooks_instance.ai_workbench_usermanaged[*].name)
-  provisioner "local-exec" {
-    command = "while [ \"$(gcloud notebooks instances list --location ${var.zone} --project ${local.project.project_id} --filter 'NAME:${each.value} AND STATE:ACTIVE' --format 'value(STATE)' | wc -l | xargs)\" != 1 ]; do echo \"${each.value} not active yet.\"; done"
-  }
+# Pod will not have access to gcloud cli
 
-  depends_on = [google_notebooks_instance.ai_workbench_usermanaged]
-}
+# resource "null_resource" "ai_workbench_usermanaged_provisioning_state" {
+#   for_each = toset(google_notebooks_instance.ai_workbench_usermanaged[*].name)
+#   provisioner "local-exec" {
+#     command = "while [ \"$(gcloud notebooks instances list --location ${var.zone} --project ${local.project.project_id} --filter 'NAME:${each.value} AND STATE:ACTIVE' --format 'value(STATE)' | wc -l | xargs)\" != 1 ]; do echo \"${each.value} not active yet.\"; done"
```

Comment:
I've got `gcloud` installed in https://github.com/PHACDataHub/sci-portal/pull/257 but haven't tested auth. Want to try that?

Reply:
I will give it a shot